content/kapacitor/v1.6/working/api.md

```diff
@@ -432,7 +438,7 @@ To get information about several tasks, make a `GET` request to the `/kapacitor/
 
 #### Example
 
-Get all tasks.
+Get all tasks. Lists details about each task, including the task ID, task type, database/retention policy (DBRP) mapping, status, whether the task is executing, and errors or stats associated with the task. Also lists the associated template ID if the task was created from a template.
```

Comment:
"Lists details..." might be more helpful as an overview of the `/tasks` response since it's common across the `/tasks` methods, not unique to `GET /tasks`. We should probably add examples here for using the query parameters.